retrieval/scrape.go

```diff
@@ -568,26 +654,63 @@ loop:
 			switch err {
 			case nil:
 			case errSeriesDropped:
+				err = nil
+				continue
+			case storage.ErrOutOfOrderSample:
+				err = nil
+				log.With("timeseries", string(met)).Warn("Out of order sample")
+				continue
+			case storage.ErrDuplicateSampleForTimestamp:
+				err = nil
+				log.With("timeseries", string(met)).Warn("Duplicate sample for timestamp")
```

Comment:
Done. I plumbed through a logging with the target labels.

I kept the per-sample logs as Debug logs.